Utils/Dataflow/pyDKB/common/hdfs.py

```diff
@@ -85,7 +128,14 @@ def movefile(fname, dest):
 def getfile(fname):
     """ Download file from HDFS.
 
-    Return value: file name (without directory)
+    Get file from HDFS and check status of process with
+    subprocess.Popen.
+
+    :param fname: path of a downloaded file
```

Comment:
"Path to the file (to download/to be downloaded)". Apart from the past tense in function arguments (see comment to line 57, point 3), it is possible to read this as "path to the local copy of the downloaded file" - which is incorrect, if I understand correctly.